src/main/java/uk/ac/ebi/biostudies/auth/User.java

```diff
@@ -28,6 +33,16 @@
     protected String email;
     protected boolean superUser;
 
+    public Set<GrantedAuthority> getAuthorities(){
+        Set<GrantedAuthority> grantedAuths = new HashSet();
+        if(allow!=null && allow.length>0)
+            for(int i=0; i<allow.length; i++)
+                grantedAuths.add(new SimpleGrantedAuthority(allow[i]));
+        if(superUser)
+            grantedAuths.add(new SimpleGrantedAuthority("SUPER_USER"));
```

Comment:
if `POWER_USER` and `SUPER_USER` are same, we should probably use a single string